src/components/SettlementButton.tsx

```diff
@@ -199,7 +215,10 @@ function SettlementButton({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [currency, formattedAmount, iouReport, policyID, translate, shouldHidePaymentOptions, shouldShowApproveButton]);
     const selectPaymentType = (event: KYCFlowEvent, iouPaymentType: PaymentMethodType, triggerKYCFlow: TriggerKYCFlow) => {
-        if (iouPaymentType === CONST.IOU.PAYMENT_TYPE.EXPENSIFY || iouPaymentType === CONST.IOU.PAYMENT_TYPE.VBBA) {
+        if (iouPaymentType === CONST.IOU.PAYMENT_TYPE.EXPENSIFY || iouPaymentType === CONST.IOU.PAYMENT_TYPE.VBBA 
+            || iouPaymentType === CONST.PAYMENT_METHODS.BUSINESS_BANK_ACCOUNT 
+            || iouPaymentType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT
+            || iouPaymentType === CONST.PAYMENT_METHODS.DEBIT_CARD) {
```

Comment:
This seems to cover every possible condition. Is there a case which is not covered by these conditions?